src/julia_threads.h

```diff
@@ -262,21 +254,17 @@ typedef struct _jl_task_t {
     // === 64 bytes (cache line)
```

Comment:
At first I though "OK, this comment doesn't make sense anymore, this is no longer offset 2*64 but rather 56+64"... Looking closer I think it actually also was wrong before?!

`JL_RNG_SIZE` is 5 so `rngState` takes up 5*8=40 bytes; the removed `metrics_enabled `and `pad1` fields another 4 bytes (but alignment rules for structs may have resulted in another 4 bytes of padding inserted by the compiler); then 3*8 for atomics; this means prior to your changes, this "section" took up 68 or even 72 bytes? So no cache line "marker" here....

Of course with your changes it is indeed now at offset 56+64, so that would suggest moving this comment down one step? Or deleting it ...